docs/testing/releases/272.md

```diff
@@ -0,0 +1,37 @@
+[![Create Todo list](https://raw.githubusercontent.com/senadir/todo-my-markdown/master/public/github-button.svg?sanitize=true)](https://git-todo.netlify.app/create)
+
+## Testing Notes and ZIP for testing
+
+**Zip File for testing:**
+[woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-gutenberg-products-block/files/4932938/woocommerce-gutenberg-products-block.zip)
+
+This release impacts the logic around draft order cleanup.
+
+Variations of tests:
+
+## When distributed as a package in WooCommerce Core.
+
+Expected behaviour here is:
+
+* [ ] `woocommerce_cleanup_draft_orders` action should not appear in the scheduled tasks for Action Scheduler (Tools->Scheduled Actions).
+
+These actions only appear when the feature flag is on for the install. If you are not testing the package inclusion in WooCommerce core, you can simulate this by adding a `.blocks.ini` file to the same directory as where this plugin is activated and include in the file the following code:
+
+```
+woocommerce_blocks_phase = 1
+```
+
+(remember to remove tha file after testing!)
+
+## When distributed as the feature plugin.
+
+Expected behaviour is:
+
+* [ ] `woocommerce_cleanup_draft_orders` action **should** appear in the scheduled tasks for Action Scheduler (Tools->Scheduled Actions).
+* [ ] Create a draft order (on the frontend load the checkout block with products in the cart and do not complete the order).
+* [ ] Trigger the scheduled action for `woocommerce_cleanup_draft_orders` and verify the just created draft order and none of the other orders on your test site have been deleted.
+* [ ] Modify the draft order date so that it's greater than 24 hours in the past from now.
```

Comment:
I needed to modify the `post_modified_gmt` column in `wp_posts` to get the order deleted.